app/services/catalog/submit_order.rb

```diff
@@ -50,6 +49,13 @@ def update_item(item, result)
       item.save!
     end
 
+    def fail_item(item)
```

Comment:
This method is for all intents and purposes duplicated a few different ways throughout the app: 
[here](https://github.com/RedHatInsights/catalog-api/blob/63fde010339cef1a81074f270974c91b8ae10e7f/app/services/catalog/create_approval_request.rb)
and [here](https://github.com/RedHatInsights/catalog-api/blob/f8fa932282944111c8d1ec42bad96347ce3e69a4/app/services/catalog/determine_task_relevancy.rb).

Though can be done in a follow up PR.